server/tox.ini

```diff
@@ -18,6 +18,7 @@ deps=
     inmlatest: inmanta-core
     inmmaster: git+https://github.com/inmanta/inmanta-core.git
     inmiso4-stable: git+https://github.com/inmanta/inmanta-core.git@iso4-stable
+    inmiso5-stable: git+https://github.com/inmanta/inmanta-core.git@iso5-stable
```

Comment:
* We need to add `iso5-stable` to `tox.envlist` as well.
* `tox.envlist` indicates that the tests run using `python3.6` and `python3.8`. I don't think that's accurate.

Reply:
Think I fixed both.